### DISCUSSION TITLE
unblock editing language

### DISCUSSION THREAD

**Discussion by alexeysvyridov** [WeblateOrg/weblate#5778] [Q&A]
I created a group with "role" translate only for Chinese, and assigned this group to component and user. Why user can't do save?

![image](https://user-images.githubusercontent.com/33163048/113116482-29041000-9216-11eb-8344-7e8c5240dbe6.png)
i found solution yet....

why should I give permission in the group "power user", to do editing of Translate? if in permissions have "Translate" ?

**Comment by tomkolp** [comment]
Do you have a project selected?
![image](https://user-images.githubusercontent.com/39633242/113296810-38f52000-92fa-11eb-828c-d2152fa40f78.png)

**Reply by alexeysvyridov** [reply]
yes i have 
![image](https://user-images.githubusercontent.com/33163048/113314342-e5dc9680-9314-11eb-8189-b238c6747880.png)
